src/index.css

```diff
@@ -51,4 +51,9 @@ td:last-child{
 
 /* td {
   word-break: break-all;
-} */
\ No newline at end of file
+} */
+
+/* Gets rids of the every other Table Row being a different color */
+td {
+  background-color: transparent !important;
+}
```

Comment:
Odd that this is needed but I'm not able to spot why the striped background is appearing.  
 
I think we should consider making this a more tightly scoped class to avoid blocking future devs from being able to add striped tables if needed. 
 
Something like `table.no-stripes > td`

Reply:
Heyo! Let's label these @shindigira with a `post-mvp` label even in the `design-system-react` repo, just so we can grab all of them easily later. 👍